### ISSUE TITLE
Decontamination

### ISSUE THREAD

**Issue by usik-luke** [usik-luke/todos#2]
Architect and create Decontamination job

Eval is the golden set

Dataset paths are here: [https://www.notion.so/trillionlabs/LUNIT-Pretraining-Plan-Nov-Dec-Plan-2aeccbda473f802299dce99900a58e24?source=copy_link#2b7ccbda473f[…]f0068d700cfb](https://www.notion.so/trillionlabs/LUNIT-Pretraining-Plan-Nov-Dec-Plan-2aeccbda473f802299dce99900a58e24?source=copy_link#2b7ccbda473f80a7a92df0068d700cfb)

**Comment by usik-luke** [comment]
`On the other hand, precisely due to the large amount of data, even GPT-3 175B does not overfit its training set by a significant amount, measured relative to a held-out validation set with which it was deduplicated (Figure 4.1). Thus, we expect that contamination is likely to be frequent, but that its effects may not be as large as feared.`

is decontamination necessary?
It is negligible

**Comment by usik-luke** [comment]
- Science 를 위해서 / data  optimization 을 위해서
- 틈틈히 하자

**Comment by usik-luke** [comment]
Olmo: https://github.com/allenai/decon
